src/embezzler.ts

```diff
@@ -106,13 +106,30 @@ export class EmbezzlerFight {
 
   /**
    * This is the class that creates all the different ways to fight embezzlers
-   * @classdesc Something goes here
+   * @classdesc Embezzler Fight enc
    * @prop {string} name The name of the source of this fight, primarily used to identify special cases.
-   * @prop {() => boolean} available Returns whether or not we can do this fight right now.
-   * @prop {() => number} potential Returns the number of embezzlers we expect to be able to fight from this source.
-   * @prop {(options: EmbezzlerFightOptions) => void} run This runs the combat, optionally using the provided location and macro. Location is used only by draggable fights.
-   * @prop {EmbezzlerFightConfigOptions} options configuration options for this fight
-   *
+   * @prop {() => boolean} available Returns whether or not we can do this fight right now (this may change later in the day).
+   * @prop {() => number} potential Returns the number of embezzlers we expect to be able to fight from this source given the current state of hte character
+   *  This is used when computing turns for buffs, so it should be as accurate as possible to the number of KGE we will fight
```

Comment:
i think this is good for a docstring, no quibbles or changes needed. but from a general "garbo dev" standpoint we might want to think about how to handle the variance we're seeing on embezzler counts vs expectation. i realize that a lot of those are bad or good luck w/ extrovermectin, but the point here is super valid; garbo may be spending a lot more on buffs than we need it to if it gets this part wrong. i realize this is a pretty vague comment but i think your garbo-debug mode would be great, and if we find that there really is a certain standard deviation on embezzler count, we may want to include that as something output by "potential" that is used in potion calculations.